src/it/java/org/checkstyle/suppressionxpathfilter/XpathRegressionUnnecessarySemicolonInTryWithResourcesTest.java

```diff
@@ -59,4 +59,28 @@ public void testOne() throws Exception {
         );
         runVerifications(moduleConfig, fileToProcess, expectedViolation, expectedXpathQueries);
     }
+
+    @Test
+    public void testAllowWhenNoBraceAfterSemicolon() throws Exception {
```

Comment:
please make names consistent, right now first test is `testOne`

Reply:
Done